src/components/OptionsList.js

```diff
@@ -62,7 +62,8 @@ const propTypes = {
     onSelectRow: PropTypes.func,
 
     /** Optional header message */
-    headerMessage: PropTypes.string,
+    // eslint-disable-next-line react/forbid-prop-types
+    headerMessage: PropTypes.object,
```

Comment:
Mentioned below I don't think we want to couple the headerMessage and rendering the SectionList but if so, you should shape the expected object instead of disabling the lint